recipes/wasmer-python/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set version = "1.1.0" %}
+
+package:
+  version: {{ version }}
+
+source:
+  url: https://github.com/wasmerio/wasmer-python/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 220af5b2675480daea4ebc1cede45728ce1b270cdc141ebff08ef0870cc23cfd
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('rust') }}
+    - cargo-bundle-licenses
+  host:
+    - python
+    - pip
+    - rustup
+    - maturin
+    - just
+    - cargo
+  run:
+    - python
+
+test:
+  imports:
+    - wasmer
+    - wasmer-compiler-cranelift
```

Comment:
these get translated into e.g.

```
python -c "import wasmer-compiler-cranelift"
```

soe the thing with `-` isn't going to import.